lib/macro/macro-action-variable.cpp

```diff
@@ -105,16 +105,20 @@ MacroActionVariable::~MacroActionVariable()
 void MacroActionVariable::HandleIndexSubString(Variable *var)
```

Comment:
I think there might still be some issues with your proposed changes.
This example action will not behave as it did previously:
![](https://github.com/user-attachments/assets/fc02ca43-8e97-43f8-ab42-32c6139ddd93)

Previously the variable would remain unchanged.
Now the first character of the current value is removed.

Reply:
I decreased wrong var by accident, should be fine now